src/components/Logo/Logo.scss

```diff
@@ -0,0 +1,9 @@
+.logo img {
```

Comment:
We should avoid using getting elements by tag name. Here you can add the className "logo" to the img, as you are only adding styles to it.